hardware/codebase/upboard/sintra/device_config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "cryptomatModel": "sintra",
+  "brain": {
+    "freeMemRatio": 0.01,
+    "powerStatus": null,
+    "dataPath": "data",
+    "wifiConfigPath": null,
+    "nfcReader": null,
+    "wifiDisabled": true
+  },
+  "kioskPrinter": {
+    "maker": "Zebra",
+    "model": "KR-403",
+    "protocol": "RS232",
+    "address": "/dev/ttyJ4"
+  },
+  "compliance": {
+    "paperWallet": false
+  },
+  "frontFacingCamera": {
+    "device": "/dev/video-front",
+    "facephoto": {
+      "width": 1280,
+      "height": 960,
```

Comment:
According to cam specs: 1280×720@30fps MJPEG; 5fps YUY2

https://www.kayetoncctv.com/non-distortion-5-megapixel-manual-focus-uvc-otg-support-usb-camera-module-5mp-ov5640-webcam-for-linux-android-windows-mac/